packages/arp/arp.0.1.1/opam

```diff
@@ -29,6 +29,11 @@ depopts: [
   "tcpip"
   "lwt"
 ]
+
+conflicts: [
+  "tcpip" {< "3.0.0"}
+]
```

Comment:
It's ok I think, the old `mirage-type` can be installed with newer `mirage-protocols` package. Seems a good feature to me!